lib/itoolkit.js

```diff
@@ -569,7 +569,29 @@ class iSql {
   }
 }
 
-
+exports.types = {
+    char(countChars) {
+        return countChars.toString() + "A";
+    },
+    packed(digits, delimiter) {
+        return digits.toString() + "P" + delimiter.toString();
+    },
+    zoned(digits, delimiter) {
+        return digits.toString() + "S" + delimiter.toString();
+    },
+    signed(digits, delimiter) {
+        return digits.toString() + "I" + delimiter.toString();
+    },
+    unsigned(digits, delimiter) {
+        return digits.toString() + "U" + delimiter.toString();
+    },

```

Comment:
AFAIK, XMLSERVICE doesn't support integer types with scale, so the delimiter must always be 0 and is thus unnecessary.

Also, I'd also like to have separate functions for each type/length that would need no parameters, eg. `signed_byte`, `signed_short`, `signed_int`, `signed_bigint`, etc..